webapp/src/locations.js

```diff
@@ -1,11 +1,12 @@
 export const locations = {
   root: '/',
 
-  address: '/address/:address',
-  addressDetail: address => `/address/${address}`,
+  profile: '/address/:address/:tab',
+  profilePage: (address, tab = 'parcels') => `/address/${address}/${tab}`,
```

Comment:
Ugh, I think we should use `TABS` here but I don't want locations to import external objects 😓 ...Maybe have a `export const PROFILE_PAGE_TABS = {}` here, and use that in the Component?